internal/commands/app/create.go

```diff
@@ -157,17 +160,13 @@ func (cmd *CommandCreate) Handler(profile *user.Profile, ui terminal.UI, clients
 		return err
 	}
 
-	if err := cmd.inputs.resolveTemplateID(ui, clients.Realm); err != nil {
+	if err = cmd.inputs.resolveTemplateID(ui, clients.Realm); err != nil {
```

Comment:
yes we do need it because we would have to fill in the --template flag if it's not already passed in by the user

I have a separate ticket for not automatically prompting for a template.

https://jira.mongodb.org/browse/REALMC-9460